packages/react-components/react-nav-preview/stories/NavCategory/NavCategoryDefault.stories.tsx

```diff
@@ -1,5 +1,22 @@
 import * as React from 'react';
-import { NavCategory } from '@fluentui/react-nav-preview';
-import type { NavCategoryProps } from '@fluentui/react-nav-preview';
+import { Nav } from '../../src/components/Nav/Nav';
+import { NavCategory } from '../../src/components/NavCategory/NavCategory';
+import { NavCategoryItem } from '../../src/components/NavCategoryItem/NavCategoryItem';
+import { NavSubItemGroup } from '../../src/components/NavSubItemGroup/NavSubItemGroup';
```

Comment:
When importing components for story files, we should should import them directly from the package. In this case since this is a preview package you have to import it from the package but later once this is stable you should import it from @fluentui/react-components. I can't remember off the top of my head why this was done, but it is related to our storybook setup.

```suggestion
import { Nav } from '../../src/components/Nav/Nav';
import { NavCategory } from '@fluentui/react-nav-preview';
import { NavCategoryItem } from '@fluentui/react-nav-preview';
import { NavSubItemGroup } from '@fluentui/react-nav-preview';
```